e107_plugins/forum/languages/English/English_notify.php

```diff
@@ -7,11 +7,29 @@
 define("LAN_FORUM_NT_9", "Forum - Thread split");
 define("LAN_FORUM_NT_10", "Forum - Post deleted");
 define("LAN_FORUM_NT_11", "Forum - Post reported");
+define("LAN_FORUM_NT_12", "Forum - Thread updated");
+define("LAN_FORUM_NT_13", "Forum - Thread moved");
+define("LAN_FORUM_NT_14", "Forum - Post created");
+define("LAN_FORUM_NT_15", "Forum - Post updated");
 
 define("LAN_FORUM_NT_NEWTOPIC", "New topic created");
 define("LAN_FORUM_NT_NEWTOPIC_PROB", "New topic created by probationary member");
-define("LAN_FORUM_NT_TOPIC_DELETED", "Topic deleted");
 define("LAN_FORUM_NT_TOPIC_SPLIT", "Topic split");
+define("LAN_FORUM_NT_TOPIC_UPDATED", "Topic updated");
+define("LAN_FORUM_NT_TOPIC_DELETED", "Topic deleted");
+define("LAN_FORUM_NT_TOPIC_MOVED", "Topic moved");
+define("LAN_FORUM_NT_POST_CREATED", "Post created");
+define("LAN_FORUM_NT_POST_UPDATED", "Post updated");
 define("LAN_FORUM_NT_POST_DELETED", "Post deleted");
 define("LAN_FORUM_NT_POST_REPORTED", "Post reported");
 
+
+define("LAN_FORUM_NT_NEWTOPIC_MSG", "New thread in forum [f] created by: [u]<br/>Subject: <a href=\"[l]\">[s]</a><br/><br/>Message:<br/>[m]<br/><br/>");
+define("LAN_FORUM_NT_NEWTOPIC_PROB_MSG", "New thread in forum [f] created by new user: [u]<br/>Subject: <a href=\"[l]\">[s]</a><br/><br/>Message:<br/>[m]<br/><br/>");
```

Comment:
Great work! Only thing is that LANs should never contain HTML, and should definitely never allow a translator to modify a link URL.  "Subject: [x]" should be enough.